libs/go/libp2p_node/dht/dhtpeer/dhtpeer.go

```diff
@@ -143,6 +143,9 @@ type DHTPeer struct {
 	closing    chan struct{}
```

Comment:
@DavidMinarsch are you planning to keep two copies of the `libp2p_node` project?

Reply:
Not permanently. But for now this is easier from a package management pov